common/src/main/scala/com/gu/media/upload/UploadsTable.scala

```diff
@@ -4,17 +4,10 @@ import com.gu.media.aws.DynamoAccess
 import com.gu.scanamo.syntax._
```

Comment:
Is there some reason why this is not called a datastore like the atom and audit datastores?